### PR TITLE
Regenerate and update

### PR DESCRIPTION
**Pull request by mattwire**
Regenerated code and cleaned up to meet latest "standards".  Marked 5.13 as min version because of security release.

### CODE REVIEW

File: CRM/Entitysetting/BAO/EntitySetting.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ static function getSettingSpecification($entity, $key = '', $force = 0) {
           }
         }
       }
-      CRM_Core_BAO_Cache::setItem($metadata, 'CiviCRM Entity setting Specs', $entity);
+      \Civi::cache()->set("entitysettings_{$entity}", $metadata);
```
Comment (seamuslee001) [inline CRM/Entitysetting/BAO/EntitySetting.php]: @mattwire what is defining this cache?

Comment (mattwire) [reply CRM/Entitysetting/BAO/EntitySetting.php]: @seamuslee001 I just changed to the new way of getting/setting cache items I think?  It seems to work, at least it caches until php execution is done - is there another way I should do it?

Comment (seamuslee001) [reply CRM/Entitysetting/BAO/EntitySetting.php]: I did this https://github.com/eileenmcnaughton/org.civicrm.multisite/blob/master/multisite.php#L64 i'm guessing your way just uses the long cache or whatever that is

Comment (seamuslee001) [reply CRM/Entitysetting/BAO/EntitySetting.php]: @mattwire the default appears to be a short cache https://github.com/civicrm/civicrm-core/blob/01aa4b6d759ee53d67144ae8cadb3e30f797003b/Civi.php#L35. Where as it currently caches it long in a specific group

Comment (eileenmcnaughton) [reply CRM/Entitysetting/BAO/EntitySetting.php]: I think the short cache is usually fine - not sure when it wouldn;t be but place for discuttion is https://github.com/civicrm/civicrm-dev-docs/issues/549

**Comment by eileenmcnaughton** [comment]
@seamuslee001 you have merge rights on this repo - leaving this to you

File: CRM/Entitysetting/BAO/EntitySetting.php (hunk 1 above)
Comment (seamuslee001) [reply CRM/Entitysetting/BAO/EntitySetting.php]: I guess the reason i raise it is its a material change as the current implementation always uses the SQLBackend which is more the 'long' cache so to speak. The question is do we care about keeping the 'CiviCRM Entity Setting Specs' group or similar.

Comment (eileenmcnaughton) [reply CRM/Entitysetting/BAO/EntitySetting.php]: @seamuslee001 I generally see this as a deprecated extension, replace by custom data on any entity - & something of an early prototype I guess. 'It still works' is a reasonable benchmark here IMHO

Comment (mattwire) [reply CRM/Entitysetting/BAO/EntitySetting.php]: @seamuslee001 I've updated the cache to use the "settings" cache per https://github.com/civicrm/civicrm-core/blob/master/Civi/Core/SettingsMetadata.php#L74 - that seems pretty much equivalent behaviour as before.

**Comment by seamuslee001** [comment]
This looks ok to me there are a couple of changes i would want to make following merge but i think its ok

**Pull request merged**